### PR TITLE
Fix linting error for helper.py

### LINKED ISSUE #647: Fix "too-many-branches" prospector error for tern/analyze/docker/helpers.py
**Description**
Fix "too-many-branches" prospector error for tern/analyze/docker/helpers.py

**To Do**
After code changes, run `prospector` (not `prospector tern/analyze/docker/helpers.py`). No prospector errors should be present for this file.

**Super Issues**
#517

### PR DESCRIPTION
**Pull request by ForgetMe17**
Add a new function get_base_image_tag to replace the deeper branches.

Fixes #647.
Signed-off-by: WangJL <hazard15020@gmail.com>

### CODE REVIEW

File: tern/analyze/docker/helpers.py [hunk 1]
```diff
@@ -57,22 +57,9 @@ def get_dockerfile_base():
     the user we will not be able to determine what the user wanted'''
     try:
         dockerfile_lines = docker_commands
```
Comment (nishakm) [inline tern/analyze/docker/helpers.py:59]: We can get rid of this line

File: tern/analyze/docker/helpers.py [hunk 2]
```diff
@@ -57,22 +57,9 @@ def get_dockerfile_base():
...
-                base_image_string = re.split(" as", cmd_dict['value'],
-                                             flags=re.IGNORECASE)[0]
-                from_line = 'FROM' + base_image_string
-                # Check that potential ARG values has default
-                if i != 0 and dockerfile_lines[i-1]['instruction'] == 'ARG':
-                    if len(dockerfile_lines[i-1]['value'].split('=')) == 1:
-                        raise ValueError('No ARG default value to pass to '
-                                         'FROM command in Dockerfile.')
-                break
+        base_image_string, from_line = get_base_image_tag(dockerfile_lines)
```
Comment (nishakm) [inline tern/analyze/docker/helpers.py:62]: And pass in `docker_commands` here

**Comment by rnjudge** [comment]
Looks good to me after Nisha's suggested changes.

**Comment by ForgetMe17** [comment]
Thanks for your review! I made some changes.

**Pull request merged**